README.md

```diff
@@ -1,198 +1,48 @@
-			* [P1] Make that workshop and talk available online
-			* [P1] Create two demos showing the power of our stack solving a “real problem”
-			* [P2] Determine what percentage of time source{d} engineers should spend working on docs/tutorials/talks/etc
-		* [P0] Make source{d} the lead of MLonCode research ([#42](https://github.com/src-d/okrs/issues/42))
-			* [P0] Create and maintain a research log with TL;DR summaries of research read
-			* [P1] Publish 6 ML blog posts in Q2
-			* [P2] Speak about ML on >4 relevant conferences
-			* [P2] Reach 1,000 cumulative upvotes of new blog posts on Reddit
-		* [P0] Enable sharing and publishing MLonCode models ([#44](https://github.com/src-d/okrs/issues/44))
-			* [P0] Define and design the product for model sharing (relates to
-			* [P1] Conduct user research with all key users (e.g.: key academic groups, users of our stack).
-		* [P0] Release updated content for sourced.tech ([#13](https://github.com/src-d/okrs/issues/13))
-			* [P0] Create a content strategy plan ([#35](https://github.com/src-d/okrs/issues/35))
-				* [P0] Editorial Environment Setup
-				* [P0] Customer Journeys
-				* [P0] Content Model
-				* [P0] Copywriting
-				* [P0] Metrics & Evaluation Analytics Setup
-				* [P1] Information Architecture
-				* [P2] Establish a Content Management Plan
-			* [P1] Update brand touchpoints according to content plan
-			* [P1] Apply new Visual Style to sourced.tech ([#32](https://github.com/src-d/okrs/issues/32))
-				* [P0] Visual Artifacts Study
-				* [P1] Research
-				* [P1] Chromatic Palete Study
-				* [P1] Typography Study
-				* [P1] Initial Proposals
-				* [P1] Pure css update PR
-			* [P1] Publish the CAT case-study development post on Medium ([#38](https://github.com/src-d/okrs/issues/38))
-				* [P0] Write a blog post regarding the process, methodologies and collaboration flow between teams on CAT
-	* Define DevRel organization ([#48](https://github.com/src-d/okrs/issues/48))
-		* [P0] Describe the metrics of devrel success
-		* [P0] Set monitoring up for all processes valuable to devrel
-		* [P1] Create a dashboard displaying those metrics
-		* [P0] Implement kanban processes in DevRel, follow the source{d} Engineering Methodology
-		* [P0] Set up biweekly sync-ups with Eiso, Marcelo, Maximo, and Fernanda
+# OKRs 2018 Q3
+
+## Bring 'Code as Data' to market with enterprises
+
+### gitbase & engine
+
+- [ ] Stable & Performance Release
+- [ ] Integration of gitbase with engine
+- [ ] New engine interface based on gitbase
+
+### Babelfish
+
+- [ ] Client/Library improvements, packaging, and stability
+- [ ] Iterate the higher level UAST abstractions to fix and maintain
+
+### Gemini
+
+- [ ] Release a new version with the integration of function similarity
+
+## Being a better company to work at
+
+- [ ] Regression Test, Benchmarks and Deploy platform (CD?)
+- [ ] Improve Logs, CLI, and other guides.
```

Comment:
Logs for Go or for Python too? CLI guide - do we have one?

Reply:
I belive this will be for all the languages that source{d} uses and is about adding a CLI guide, so we will have one, with best practices documented, pushing further the effort that @smola has already started this quarter on:
 - Configuration https://github.com/src-d/guide/pull/214
 - Error handling https://github.com/src-d/guide/pull/213

Reply:
@vmarkovtsev Go, Python, Scala

We do have some basics in src-d/guide (merged and/or PRs), and some more specifics will come up, modeled after https://github.com/src-d/borges/pull/294